### PR TITLE
Add icons

### PR DESCRIPTION
**Pull request by kimoshinowo98**
Ticket: https://webjobs.agepartnership.co.uk/desk/tickets/6599885/messages

Changes:
- Add remaining icons from Image library

Checks:
- Non-malicious
- Appropriate names

### CODE REVIEW

File: rebuild/assets/images/icons/calculator--primary.svg
Comment (beccadawson) [inline rebuild/assets/images/icons/calculator--primary.svg:1]: This seems black not blue

**Review by beccadawson — COMMENTED** [review]
Few of these seem black not blue and a couple seem not-transparent, also wanted to check how you got the SVGs as I think the icons in the library were pngs?

**Comment by kimoshinowo98** [comment]
@beccadawson I used a couple of different online converters (one would only let me convert a certain number so had to try some others)

**Comment by beccadawson** [comment]
> @beccadawson I used a couple of different online converters (one would only let me convert a certain number so had to try some others)

Will be better off getting the originals (because reasons 😅), I'll ask ATP

**Review by beccadawson — APPROVED** [review]

**Pull request merged**